content/tutorials/history.textile

```diff
@@ -355,6 +355,19 @@ blang[ruby].
 
   And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @example.rb@ and running the demo @bundle exec ruby example.rb@
 
+blang[php].
+  The complete source code for each step of "this tutorial is available on Github":https://github.com/ably/tutorials/commits/history-ruby.
+
+  We recommend that you clone the repo locally:
+
+  bc[sh]. git clone https://github.com/ably/tutorials.git
+
+  Checkout the tutorial branch:
+
+  bc[sh]. git checkout history-php
+
+  And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @index.php@ and running this script with @php index.php@.
```

Comment:
Again no reference to `public` folder used in the tutorial code.  I'd prefer if we changed the tutorial to not place the file in `public` anyway as it seems unnecessary given the simplicity of what we're doing.